.github/workflows/test_jax_dockerfile.yml

```diff
@@ -6,7 +6,7 @@ on:
       test_runs_on:
         required: true
         type: string
-        default: "linux-mi325-1gpu-ossci-rocm-frac"
+        default: "linux-mi325-1gpu-ossci-rocm"
```

Comment:
Is this change intended? Are the fractional runners sufficient for this testing? (I would expect them to be)